app/views/time_clock_punches/_time_clock_punch.html.erb

```diff
@@ -0,0 +1,18 @@
+<tr id="<%= dom_id time_clock_punch %>">
+  <td><%= link_to time_clock_punch.person.display_name, time_clock_punch.person, class: 'btn btn-primary', 'data-turbo': false %></td>
```

Comment:
Just do `as_badge(time_clock_punch.person)` here instead - although that will need hacked on to account for the fact that you want to skip turbo here.